src/library/scala/collection/ArrayOps.scala

```diff
@@ -491,8 +491,15 @@ final class ArrayOps[A](val xs: Array[A]) extends AnyVal {
       case xs: Array[AnyRef] =>
         val a = Arrays.copyOf(xs, len); Arrays.sort(a, ord.asInstanceOf[Ordering[AnyRef]]); a
       case xs: Array[Int] =>
-        if(ord eq Ordering.Int) { val a = Arrays.copyOf(xs, len); Arrays.sort(a); a }
-        else boxed
+        println("in array pattern matching. Matched as Array ofInt")
```

Comment:
add a test instead of println:)

Reply:
Sorry about that. Should I remove it and send a pull request again?

Reply:
nah, you can just force-push the changes

Reply:
Commit,rebase,pick and fix up,push -f

Reply:
@NthPortal @hepin1989 done

Reply:
@hepin1989 Who would be merging this pull request?

Reply:
@ashwinbhaskar Teamember

Reply:
@hepin1989 @NthPortal @adriaanm Can I know how many LGTMs before merge happens? Just curious and trying to understand how things go about here:) This is my first contribution to scala core lib